hpy/universal/src/hpymodule.c

```diff
@@ -155,7 +155,7 @@ static PyObject *get_version(PyObject *self, PyObject *ignored)
 }
 
 static PyMethodDef HPyMethods[] = {
-    {"load", (PyCFunction)load, METH_VARARGS | METH_KEYWORDS, "Load a .hpy.so"},
+    {"load", (PyCFunction)load, METH_VARARGS | METH_KEYWORDS, "Load a .hpy0.so"},
```

Comment:
We are also using this in `hpy/devel/abitag.py`. I think these two usages should have a strong connection. Can we do what we do with the version? Define it in a C header and expose it in Python.

Reply:
One little more thing: there is an `int ctx_version` field in `HPyContext`. Matti brought to my attention, that this field basically is the ABI version, right? So, I think we should set `ctx_version` appropriately.

Reply:
yes, good point. It turns out that sharing the same definition in C and Python is a mess as usual, so I opted for defining it in two places (hpy.h and abitag.py) and add a test which checks that they are on sync.
Also, I renamed `ctx->ctx_version` into `ctx->abi_version` and set its value to `HPY_ABI_TAG`, see commits 71d2a7d and 1f70e65.